CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v0.2.6 -
+
+## Changed
+
+- #127: Increase default `raw_notification_size` from 1000 to 8192, and also update this and `raw_notification_initial_days_since` definition in `PLUGINS_CONFIG`.
```

Comment:
Given this is a backwards-incompatible change in the plugin configuration, should we treat this commit as beginning a 0.3.0 release rather than a 0.2.x patch release?

Reply:
good point